src/writeData/components/WriteDataItem.tsx

```diff
@@ -36,7 +37,11 @@ const WriteDataItem: FC<Props> = ({id, name, url, image, history, orgID}) => {
   let thumb = <img src={placeholderLogo} />
 
   if (image) {
-    thumb = <img src={image} />
+    thumb = (
+      <Suspense fallback="Loading...">
```

Comment:
I tried using the `placeholderLogo` as the `fallback` rather than text but it actually never appears because the request to load the placeholder svg gets lost amongst the request to load the real svgs, thereby not improving the situation at all lol.

Reply:
https://reactjs.org/docs/concurrent-mode-suspense.html

>This page describes experimental features that are not yet available in a stable release. Don’t rely on experimental builds of React in production apps. These features may change significantly and without a warning before they become a part of React.

is this something we want going out in prod?

>This documentation is aimed at early adopters and people who are curious. If you’re new to React, don’t worry about these features — you don’t need to learn them right now. For example, if you’re looking for a data fetching tutorial that works today, read [this article](https://www.robinwieruch.de/react-hooks-fetch-data) instead.

Reply:
that comment applies to suspense fetching data, not the component being used here